javascript-data-model/order-history.js

```diff
@@ -0,0 +1,93 @@
+var orderHistory = [
+  [
+    {
+      orderNumber: 11428755,
+      total: 44.53,
+      orderPlacedDate: 'July 19, 2020',
+      deliveryDate: 'Delivered Jul 20 2020',
+      shipTo: 'JS Masher',
+      deliveredStatus: true,
+      returnWindow: 'Aug 19',
+      buyAgain: '',
+      orderDetails: '',
+      item: {
+        type: 'book',
+        price: 41.33,
+        invoice: '',
+        writeProductReview: '',
+        returnWindow: 'Sept7, 2020',
+        additionalInfo: {
+          author: 'Alexander, Christopher',
+          title: 'The timeless Way of Building',
+          productReview: '',
+          viewItem: ''
+        }
+      }
+    }
+  ],
+  [
+    {
+      orderNumber: 11428757,
+      total: 17.22,
+      orderPlacedDate: 'July 4, 2020',
+      deliveryDate: 'Delivered Jul 7, 2020',
+      shipTo: 'JS Masher',
+      deliveredStatus: true,
+      returnWindow: 'Aug 19, 2020',
+      orderDetails: '',
+      item: {
+        type: 'computer parts',
+        price: 15.98,
+        invoice: '',
+        returnWindow: 'Aug 19',
+        additionalInfo: {
+          name: 'gamecube controllelr',
+          brand: 'Nintendo',
+          productReview: '',
+          buyAgain: '',
+          viewItem: ''
+        }
+      }
+    }
+  ],
+  [
+    {
+      orderNumber: 1132883177,
+      total: 138.93,
+      orderPlacedDate: 'July 3, 2020',
+      deliveryDate: 'Delivered Jul 5, 2020',
+      shipTo: 'JS Masher',
+      deliveredStatus: true,
+      buyAgain: '',
+      orderDetails: '',
+      invoice: '',
+      item: [
```

Comment:
But in this order, `item` is an array. That is not consistent and would lead to errors. You will need to make all objects use the same format in order for the data to be usable.